CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ endif()
 # =====
 
 set(XTENSOR_SPARSE_HEADERS
-    ${XTENSOR_SPARSE_INCLUDE_DIR}/xtensor-sparse/xlil.hpp
+    ${XTENSOR_SPARSE_INCLUDE_DIR}/xtensor-sparse/xmap.hpp
```

Comment:
Is that possible to name the file `xmap_container`? `xmap` sounds like a specific implementation of a standard map.